STM32F405RGTx/Core/Src/bms_entry.cpp

```diff
@@ -24,20 +24,16 @@ uint8_t __io_getchar() {
 BMS global_bms_data;
 Ltc6813 ltc6813;
 
-osThreadId_t measurements_thread;
-osThreadId_t coulomb_counting_thread;
-osThreadId_t state_machine_thread;
-
-void _lv_test_init_global_bms_obj() {
-    global_bms_data.mc_cap_voltage = 46;
-    global_bms_data.contactor_voltage = 46;
-    global_bms_data.buck_temp = 25;
-    global_bms_data.battery.voltage = 46;
-    global_bms_data.battery.current = 15;
-    global_bms_data.battery.soc = 100;
+void BMS::_lv_test_init() {
```

Comment:
this function doesn't need to be here, it initializes the data to nominal values. Instead, maybe we can just let the measurements thread run for a while, before starting the state machine so it doesn't immediately throw a fault.